src/components/ComponentListView/ComponentListViewPage.scss

```diff
@@ -0,0 +1,7 @@
+.hacDev-component-list-view-page__trash-icon {
+  color: var(--pf-global--BackgroundColor--dark-400);
+  &:hover {
+    cursor: pointer;
```

Comment:
`<Button>` doesn't make it pointer by default?

Reply:
Thanks for catching this. Removed the css